### PR TITLE
Ret 5

### PR DESCRIPTION
**Pull request by djdavies**
These are all the migrations from the whiteboard session. Please review.

### CODE REVIEW

File: database/migrations/2016_12_06_110644_create_industries_table.php [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('industries', function(Blueprint $table) {
+            $table->increments('id');
+            $table->string('name')->unique();
+            $table->rememberToken();
```
Comment (Tanprasit) [inline database/migrations/2016_12_06_110644_create_industries_table.php:19]: Doesn't need the remember token as it is used for logins

File: database/migrations/2016_12_06_123758_create_learnings_table.php [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('learnings', function(Blueprint $table) {
+            $table->increments('id');
+            $table->string('name');
+            $table->date('dateStarted');
```
Comment (Tanprasit) [inline database/migrations/2016_12_06_123758_create_learnings_table.php:19]: Change to underscore

File: database/migrations/2016_12_06_124120_create_skill_user_table.php [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+class CreateSkillUserTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('skill_user', function(Blueprint $table) {
```
Comment (Tanprasit) [inline database/migrations/2016_12_06_124120_create_skill_user_table.php:16]: Add an id to this table

File: database/migrations/2016_12_06_124241_create_classification_skill_table.php [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('classification_skill', function(Blueprint $table) {
+            $table->integer('classification_id');
```
Comment (Tanprasit) [inline database/migrations/2016_12_06_124241_create_classification_skill_table.php:17]: add id here

File: database/migrations/2016_12_06_124404_create_classification_learning_table.php [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+class CreateClassificationLearningTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('classification_learning', function(Blueprint $table) {
```
Comment (Tanprasit) [inline database/migrations/2016_12_06_124404_create_classification_learning_table.php:16]: Add another id here

**Review by Tanprasit — CHANGES_REQUESTED** [review]
Make the changes please

File: database/migrations/2016_12_06_124120_create_skill_user_table.php (hunk 3 above)
Comment (djdavies) [reply database/migrations/2016_12_06_124120_create_skill_user_table.php:16]: Nope: http://stackoverflow.com/questions/34274998/any-advantages-of-adding-an-id-column-to-a-pivot-table-in-laravel

**Review by djdavies — COMMENTED** [review]
All done, but not added ids into pivots as discussed.

**Pull request merged**